src/PhoneNumber.php

```diff
@@ -3,7 +3,7 @@
 namespace Bissolli\NovaPhoneField;
 
 use Laravel\Nova\Fields\Field;
-
+use Illuminate\Support\Arr;
```

Comment:
Can you break like here so it will pass through our StyleCI?

```suggestion
use Illuminate\Support\Arr;

```